CHANGES.md

```diff
@@ -1,10 +1,18 @@
-## 0.x.y
+## 0.8.0
 
 * Allow routers to be configured with a list of identifiers.  If an identifier
   cannot assign a dest to a request, it falls back to the next one in the list.
   * **Breaking Change**: Identifier plugins must now return a
     `RequestIdentification` object.
+* Consul improvements:
+  * Improve performance by only watching services as necessary and tearing
+    down watches when they are no longer needed.
+  * Add `consistencyMode` option to `io.l5d.consul` namer
+  * Add `readConsistencyMode` and `writeConsistencyMode` options to `io.l5d.consul` dtab storage
+  * Consul Namerd/DtabStore: `failFast` and `failureAccrual` is now
+    disabled by default but can be enabled with the `failFast` option
 * Improve shutdown ordering to facilitate graceful shutdown.
+  * Gracefully shutdown on SIGINT and SIGTERM.
 * Require tracer configuration instead of falling back to
   defaults, reducing logging noise.
 * The `debugTrace` tracer configuration flag has been removed in favor
```

Comment:
I don't think it breaks anything. It's an improvement. SIGINT and SIGTERM would shut down the process before, just not before draining its queues.  In other words, I don't think we would have bumped minor version for this change alone.